UnityProject/Assets/Scripts/Managers/EventManager.cs

```diff
@@ -30,7 +30,8 @@ public enum Event
 	PreRoundStarted,
 	MatrixManagerInit,
 	BlobSpawned,
-	ScenesLoadedServer
+	ScenesLoadedServer,
+	LavalandFirstEntered
```

Comment:
would be nice if it wasn't hardcoded but I can't think of any way where it wouldn't be?